docs/DeveloperGuide.md

```diff
@@ -689,18 +695,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
    Use case ends.
 
 **Extensions**
-* 2a. The update data is invalid.
+* 2a. The guest's bill is negative.
 
     * 2a1. System shows an error message.
 
-      Use case resumes at step 2.
+      Use case ends.
+
+* 2b. The update data is invalid.
```

Comment:
Should we say "The value provided is invalid"?

Reply:
Agreed!